src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -148,15 +148,27 @@
            [react/view {:style (message.style/delivery-status outgoing)}
             [message/message-delivery-status message]]])}))))
 
+(def load-step 5)
+
+(defn load-more [messages messages-to-load]
+  (let [all-messages-count (count messages)
+        next-count (min all-messages-count (+ @messages-to-load load-step))]
+    (reset! messages-to-load next-count)))
+
 (views/defview messages-view [{:keys [chat-id group-chat]}]
   (views/letsubs [messages [:get-current-chat-messages-stream]
-                  current-public-key [:get-current-public-key]]
-    (let [chat-id* (atom nil)
-          scroll-ref (atom nil)
+                  current-public-key [:get-current-public-key]
+                  messages-to-load (reagent/atom 5)
+                  chat-id* (reagent/atom nil)]
```

Comment:
messages-count could be binded there instead of recalculating it every time in load-more

Reply:
Do you mean creating atom outside defview?

Reply:
no creating it here `message-count (count @messages)`